src/deployment/azuredeploy.bicep

```diff
@@ -0,0 +1,677 @@
+    sku: {
+      family: 'A'
+      name: 'standard'
+    }
+    networkAcls: {
+      defaultAction: 'Allow'
+      bypass: 'AzureServices'
+    }
+    tenantId: tenantId
+    accessPolicies: [
+      {
+        objectId: reference(resourceId('Microsoft.Web/sites', name), '2019-08-01', 'full').identity.principalId
+        tenantId: tenantId
+        permissions: {
+          secrets: [
+            'get'
+            'list'
+            'set'
+            'delete'
+          ]
+        }
+      }
+    ]
+  }
+}
+
+resource serverFarms 'Microsoft.Web/serverfarms@2021-03-01' = {
+  name: name
+  location: location
+  kind: 'linux'
+  properties: {
+    reserved: true
+  }
+  sku: {
+    name: 'P2v2'
+    tier: 'PremiumV2'
+    family: 'Pv2'
+    capacity: 1
+  }
+  tags: {
+    OWNER: owner
+  }
+}
+
+resource autoscaleSettings 'Microsoft.Insights/autoscalesettings@2015-04-01' = {
+  name: autoscale_name
+  location: location 
+  properties: {
+    name: autoscale_name
+    enabled: true
+    targetResourceUri: serverFarms.id
+    targetResourceLocation: location
+    notifications: []
+    profiles:[
+      {
+        name: 'Auto scale condition'
+        capacity: {
+          default: '1'
+          maximum: '20'
+          minimum: '1'
+        }
+        rules: [
+          {
+            metricTrigger: {
+              metricName: 'CpuPercentage' 
+              metricResourceUri: serverFarms.id
+              operator: 'GreaterThanOrEqual'
+              statistic: 'Average'
+              threshold: 20
+              timeAggregation: 'Average'
+              timeGrain: 'PT1M'
+              timeWindow: 'PT1M'
+            }
+            scaleAction: {
+              cooldown: 'PT1M'
+              direction: 'Increase'
+              type: 'ChangeCount'
+              value: '5'
+            }
+          }
+          {
+            metricTrigger: {
+              metricName: 'CpuPercentage'
+              metricResourceUri: serverFarms.id
+              operator: 'LessThan'
+              statistic: 'Average'
+              threshold: 20
+              timeAggregation:'Average' 
+              timeGrain: 'PT1M'
+              timeWindow: 'PT1M'
+            }
+            scaleAction: {
+              cooldown: 'PT5M'
+              direction: 'Decrease'
+              type: 'ChangeCount'
+              value: '1'
+            }
+          }
+
+        ]
+      }
+    ]
+  }
+  tags: {
+    OWNER: owner
+  }
+}
+
+resource insightsWorkspace 'Microsoft.OperationalInsights/workspaces@2021-06-01' = {
+  name: name
+  location: location
+}
+
+var linuxDataSources = [
+  {
+    name: 'syslogDataSourcesKern'
+    syslogName: 'kern'
+    kind: 'LinuixSyslog'
+  }
+  {
+    name: 'syslogDataSourcesUser'
+    syslogName: 'user'
+    kind: 'LinuixSyslog'
+  }
+  {
+    name: 'syslogDataSourcesCron'
+    syslogName: 'cron'
+    kind: 'LinuixSyslog'
+  }
+  {
+    name: 'syslogDataSourcesDaemon'
+    syslogName: 'daemon'
+    kind: 'LinuixSyslog'
```

Comment:
Typo?
```suggestion
    kind: 'LinuxSyslog'
```

Reply:
There were a few other other ones in this same block, sorry I only highlighted one.

Reply:
yup, fixed those just now